sci-mathematics/glucas/glucas-2.9.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A client to make primality test for Mersenne numbers"
+DESCRIPTION="Glucas is related to GIMPS effort to search the biggest primes ever found."
+HOMEPAGE="https://sourceforge.net/projects/glucas/"
+COPYRIGHT="gbvalor"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/glucas/Glucas-$portVersion.tar.gz"
+CHECKSUM_SHA256="fde422220245788c0d8aeaec6a89f26e849905b1417c5e6ff1a01f56c595cadc"
+#PATCHES=""
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	glucas = $portVersion
+	cmd:Glucas = $portVersion
+		"
+REQUIRES="
+	haiku
+		"
+		
+BUILD_REQUIRES="
+	haiku_devel
+		"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc
+	cmd:make
+		"
+		
+BUILD()
+{
+	autoreconf -fi
+	./configure --prefix="$prefix"  --infodir="$infoDir" --disable-dependency-tracking
```

Comment:
Regarding the error "makeinfo: command not found", we could add a `cmd:makeinfo` but it is better to add a `MAKEINFO=true`:
```diff
 BUILD()
 {
 	autoreconf -fi
-	./configure --prefix="$prefix"  --infodir="$infoDir" --disable-dependency-tracking
+	MAKEINFO=true \
+	./configure --disable-dependency-tracking \
+		--prefix="$prefix" --infodir="$infoDir"
 	make $jobArgs
 }
```
(We like to have lines <= 80 chars.)

Reply:
runConfigure should be used instead of prefix and infodir

Reply:
Try this instead:
```diff
-	./configure --prefix="$prefix"  --infodir="$infoDir" --disable-dependency-tracking
+	runConfigure ./configure --disable-dependency-tracking
```
runConfigure will take care of the needed paths for prefix, infodir ...